<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeSendWdg</class>
 <widget class="QWidget" name="NoticeSendWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>发布公告</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="vCLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{ 
	
/*border-radius:10px; */
	background-color: rgb(52, 54, 58);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget{
border: 0px;
	background-color: rgb(64, 66, 72);
}

QListWidget:item{
	width:320px;
	border-radius: 0px;
}
QScrollBar:vertical
{
background:rgb(0,0,0);
border-radius:4px;
width:6px;
margin:0px 0px 0px 0px;

}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{

background-color:  rgba(64, 66, 72);
border: 0px;
}

QScrollBar::handle:vertical
{
background:rgba(255,255,255);
border-radius:3px;
min-height:10px;
border: 1px solid #D2D2D2;
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_send" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>52</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2{
	background-color: rgb(52, 54, 58);
/*border-bottom-right-radius :10px; 
border-bottom-left-radius :10px; */
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>11</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QTextEdit {	
font: 12px &quot;微软雅黑&quot;;
	border: 1px solid rgb(196, 196, 196);
	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px;
    background: white;
	padding-left: 10px;
    color: rgb(90, 90, 90);
}
 
QTextEdit:hover {	
	border: 1px solid rgb(153, 153, 153);
  	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px; 
    background: white;
    color: rgb(254, 99, 106);
}

QTextEdit:focus {	
	border: 1px solid rgb(153, 153, 153);
 	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px;
    background: white;
	padding-left: 10px;
    color: rgb(90, 90, 90);
}

QScrollBar:vertical
{
border-radius:4px;
width:6px;
background:rgba(255,255,255, 255);
margin:0px,0px,0px,0px;
overflow-x:hidden;
}

QScrollBar::handle:vertical
{
background:rgba(210,210,210, 255);
border-radius:3px;
min-height:20;
border: 1px solid #D2D2D2;
}</string>
           </property>
           <property name="undoRedoEnabled">
            <bool>false</bool>
           </property>
           <property name="acceptRichText">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>请输入公告~</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_send">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
font: 12px   &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;
	background-color: rgb(166, 166, 168);
}

QPushButton:hover{
font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;
	background-color: rgb(150, 150, 150);
}

QPushButton:disable{
font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;	
	background-color: rgb(180, 180, 180);
}

</string>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
